tests/test.validator.js

```diff
@@ -476,3 +477,199 @@ describe('Validator.textOutput()', function() {
     }
   });
 });
+
+
+describe('Validator.detectType()', function() {
+
+  it('should reject on multiple em:type nodes', () => {
+    var addonValidator = new Validator({_: ['bar']});
+    addonValidator.xpi = {
+      getFileAsString: () => {
+        return Promise.resolve(singleLineString`
+          <?xml version='1.0' encoding='utf-8'?>
```

Comment:
There's a `validRDF()` helper in [`tests/helpers.js`](https://github.com/mozilla/addons-validator/blob/master/tests/helpers.js#L22) that reduces the boilerplate in these tests. Instead of this whole string you just need:

```js
validRDF(singleLineString`
<em:type>2</em:type>
<em:type>2</em:type>
`)
```